src/main/resources/assets/admin/common/js/ui/form/Validators.ts

```diff
@@ -5,20 +5,31 @@ import {StringHelper} from '../../util/StringHelper';
 export class Validators {
 
     public static required(input: FormInputEl): string {
-        let value = input.getValue();
-        return StringHelper.isBlank(value) ? i18n('field.value.required') : undefined;
+        return StringHelper.isBlank(input.getValue()) ? i18n('field.value.required') : undefined;
     }
 
     public static validEmail(input: FormInputEl): string {
-        let regexEmail = /\w+([-+.']\w+)*@\w+([-.]\w+)*\.\w+([-.]\w+)*/;
-        let value = input.getValue();
-        return !regexEmail.test(value) ? i18n('field.value.invalid') : undefined;
+        const regex = /\w+([-+.']\w+)*@\w+([-.]\w+)*\.\w+([-.]\w+)*/;
+        return !regex.test(input.getValue()) ? i18n('field.value.invalid') : undefined;
     }
 
-    public static validUrl(input: FormInputEl): string {
-        let regexUrl =
-            /^http(s)?:\/\/\w+(\.\w+)*(:[0-9]+)?(\/.*)?$/;
-        let value = input.getValue();
-        return !regexUrl.test(value) ? i18n('field.value.invalid') : undefined;
+    public static validHttps(input: FormInputEl): string {
+        const regex = /^https:\/\/\w+(\.\w+)*(:[0-9]+)?((\/|\#).*)?(\?([^&=]+)=([^&=]+))?(?:&([^&=]+)=([^&=]+))*$/;
+        return !regex.test(input.getValue()) ? i18n('field.value.invalid') : undefined;
+    }
+
+    public static validHttp(input: FormInputEl): string {
+        const regex = /^http:\/\/\w+(\.\w+)*(:[0-9]+)?((\/|\#).*)?(\?([^&=]+)=([^&=]+))?(?:&([^&=]+)=([^&=]+))*$/;
+        return !regex.test(input.getValue()) ? i18n('field.value.invalid') : undefined;
+    }
```

Comment:
Actually, I'm a bit sceptical to having two different methods for validating http and https. The regexps are identical (the only difference is the protocol) and it's too easy to make a mistake by changing the regexp in just one of the methods.

I think we should keep one method, as before (with a regexp that starts with `http(s)` prefix).